extensions/indexes/lucene/src/main/java/org/exist/xquery/modules/lucene/Facets.java

```diff
@@ -75,7 +78,7 @@
                     "to the entire sequence returned by ft:query, so the same map will be returned for all nodes in the sequence. " +
                     "It is thus sufficient to specify one node from the sequence as first argument to this function.",
             new SequenceType[] {
-                    new FunctionParameterSequenceType("node", Type.NODE, Cardinality.EXACTLY_ONE,
+                    new FunctionParameterSequenceType("nodes", Type.NODE, Cardinality.ZERO_OR_MORE,
                             "A single node resulting from a call to ft:query for which facet information should be retrieved. " +
```

Comment:
Again the description needs updating. Not sure about the cardinality (as above).